### DISCUSSION TITLE
Validate URL

### DISCUSSION THREAD

**Discussion by itrussell15** [RIAEvangelist/node-dominos-pizza-api#107] [Q&A] [answered]
After browsing through the url endpoints, I stumbled across the "validate order url"... Just to be sure, this will take an order and ensure that it is acceptable to be processed? I am just at the point of wanting to post my order using the place order url, but don't want to order 20 pizzas in the process, so this will help me know if the final order will be correct or not if it going to do what I think it is.

Thanks in advance.

**Comment by RIAEvangelist** [comment, marked as answer]
Correct. This is the URL used by the `await order.validate();` call.

An examples/documentation can be found in the main read me as well as in the [docs/Order.md](https://github.com/RIAEvangelist/node-dominos-pizza-api/blob/master/docs/Order.md)
